src/org/usfirst/frc/team2335/robot/commands/AutoTurn.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+//TODO: Fine tune values
+
+public class AutoTurn extends Command
+{
+	private double _turnVal;
+	
+    public AutoTurn(int turnVal) 
+    {
+        _turnVal = turnVal > 0 ? 0.3 : -0.3;
```

Comment:
What does this logic mean for _turnVal? More specifically, what does the ? and : do in the code?